bugRzilla_review.Rmd

```diff
@@ -26,22 +26,21 @@ body {
 
 ## What is the bugRzilla Package?
 
-BugRzilla is an R package that helps the user to interact with the **Bugzilla** through an API.
+The bugRzilla is an R package that helps the user to interact with the **Bugzilla** through an API. Bugzilla, a bug-tracking system that enterprise-class piece of software that tracks millions of bugs and issues for thousands of organizations around the world.
 
-<font size="3">To learn more, see [bugRzilla](https://github.com/llrs/bugRzilla).</font>
+<font size="3">The source code for this report is available in the [bugRzilla](https://github.com/llrs/bugRzilla) GitHub repository.</font>
```

Comment:
Differentiate between the report and the package! on https://github.com/llrs/bugRzilla there is the package, the report is on https://github.com/llrs/bugzilla_viz as you correctly explain some lines below.

Reply:
Sure working on it.